usr.sbin/devinfo/devinfo.c

```diff
@@ -78,13 +78,22 @@ print_resource(struct devinfo_res *res)
 {
 	struct devinfo_rman	*rman;
 	bool			hexmode;
+	rman_res_t		end;
```

Comment:
Don't know how to add a comment on the commit message...

But this change looks good, I'd say 'unwind the ternary operator to better fit future libxo structure. No functional change intended.' instead of just no functional change intended.

Reply:
Fixed.